### DISCUSSION TITLE
Permission about changing some code

### DISCUSSION THREAD

**Discussion by StarsationX** [oobabooga/text-generation-webui#6543] [Q&A]
Hello, I'm here to ask if I can change, add some of the code

I'm doing my School Final Project and my teacher asked me if I can add like color palette things or theme and a media player because I use text-generation-webui to host an Ai that help with depression (well I got this idea when I was using C.AI) and since I cant just change some of the code and just send it to her

My idea is well the user have to download all of the thing that'll require to run text-generation-webui locally, I'm here to ask for your permission

This is my first time doing things like this so I'm sorry in advance if I've done anything wrong 🙏

**Comment by TheLounger** [comment]
You can create a [fork](https://github.com/oobabooga/text-generation-webui/fork), change whatever you want, and let others download *your* version. You can also make a Pull Request if you think your changes will add value for many people. You still need to make a fork before you can make a PR, however.

**Reply by StarsationX** [reply]
ohhhhhhhhhhhhhh thank youuuu
